financialaid/views.py

```diff
@@ -218,3 +223,20 @@ def get_queryset(self):
         )
 
         return financial_aids
+
+
+class FinancialAidActionView(UpdateAPIView):
+    """
+    View for rejecting and approving financial aid requests
+    """
+    serializer_class = FinancialAidActionSerializer
+    permission_classes = (IsAuthenticated, UserCanEditFinancialAid)
+    lookup_field = "id"
+    lookup_url_kwarg = "financial_aid_id"
+    queryset = FinancialAid.objects.all()
+
+    def post(self, request, *args, **kwargs):
```

Comment:
We just put it in for flexibility, but you're right it's not required.  We can take it out if you'd like?